src/scenes/GameScene.tsx

```diff
@@ -13,11 +14,19 @@ export default class GameScene extends Phaser.Scene {
   private gameTimeText: Phaser.GameObjects.Text | undefined;
   private eventSubscription: (() => void) | undefined;
 
+  private worldModel: WorldModel;
+  private worldView: WorldView;
+
   public constructor() {
     super({ key: 'GameScene' });
+
+    this.worldModel = new WorldModel(0, 30, 30);
```

Comment:
If these are hard-set we could put them in a constants.ts file - even if temporary, same with renderConfig later

eg. import { WORLD_WIDTH } from './../../constants.ts;

just helps edit game stats all in one place - it was very helpful for my tower game https://github.com/GeorgeHarland/RandomTower/blob/main/src/constants.ts